suite-native/module-settings/src/components/ColorSchemePicker.tsx

```diff
@@ -18,7 +18,6 @@ export const ColorSchemePicker = () => {
             <Text>Color Scheme</Text>
             <HStack style={applyStyle(stackStyle)} spacing="small">
                 <ColorSchemePickerItem colorScheme="standard" />
-                <ColorSchemePickerItem colorScheme="chill" />
```

Comment:
I would replace this with system ColorSchemePickerItem so they are all in one line. Only reason the system one was placed below with full width is that 4 in line was too much. Maybe double-check with @mnuky